thoth/storages/sync.py

```diff
@@ -61,8 +63,13 @@ def sync_adviser_documents(
         adviser_store = AdvisersResultsStore()
         adviser_store.connect()
 
+    listing = list(adviser_store.get_document_listing()) or document_ids
```

Comment:
Can we cast the generator to a list only if randomization is on? I know it might be nitpicking, but in some cases it might improve perf :)

Reply:
No you're right, thanks! :100: Added in https://github.com/thoth-station/storages/pull/2625/commits/03acd31c136b92423ef836ac38869fed6fd5e13c